ui/raidboss/data/05-shb/trial/castrum-ex.js

```diff
@@ -0,0 +1,123 @@
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Divide Et Impera P1',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '555B', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'CastrumEx Divide Et Impera P2',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '5537', capture: false }),
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '八方位',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Primus Terminus Est',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55C3', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '靠边',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Tertius Terminus est',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55CC', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '注意落剑顺序',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Sidescathe Left',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55D5', capture: false }),
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '左(看翅膀)',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Full-Power Optimized Ultima',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '5B17', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '凉凉',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Sidescathe Right',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55D4', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '右（看翅膀）',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Secundus Terminus est',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55C8', capture: false }),
+      alarmText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '靠边放剑(十字四角)',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Legio Phantasmatis',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55B4', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '注意士兵顺序，结束返回中央',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Magitek Magnetism',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '555B', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '注意地雷正负极(正正扩散，正负融合)',
```

Comment:
I am not sure that this explanation is really enough to do the mechanic.  It sounds like tanks need to do something different here too? Maybe this is one that's better to remove for now and then done differently with headmarker and tether lines.

Reply:
right, it seems I'm struggling to find anything else related to this mechanism. Maybe someone else can add it in later.